src/mvAppItem.cpp

```diff
@@ -4475,14 +4623,27 @@ DearPyGui::GetEntityParser(mvAppItemType type)
-        args.push_back({ mvPyDataType::Bool, "log_scale", mvArgType::KEYWORD_ARG, "False" });
-        args.push_back({ mvPyDataType::Bool, "invert", mvArgType::KEYWORD_ARG, "False" });
-        args.push_back({ mvPyDataType::Bool, "lock_min", mvArgType::KEYWORD_ARG, "False" });
-        args.push_back({ mvPyDataType::Bool, "lock_max", mvArgType::KEYWORD_ARG, "False" });
-        args.push_back({ mvPyDataType::Bool, "time", mvArgType::KEYWORD_ARG, "False" });
+        args.push_back({ mvPyDataType::Bool, "no_label", mvArgType::KEYWORD_ARG, "False", "the axis label will not be displayed" });
+        args.push_back({ mvPyDataType::Bool, "no_gridlines", mvArgType::KEYWORD_ARG, "False", "no grid lines will be displayed" });
+        args.push_back({ mvPyDataType::Bool, "no_tick_marks", mvArgType::KEYWORD_ARG, "False", "no tick marks will be displayed" });
+        args.push_back({ mvPyDataType::Bool, "no_tick_labels", mvArgType::KEYWORD_ARG, "False", "no text labels will be displayed" });
+        args.push_back({ mvPyDataType::Bool, "no_initial_fit", mvArgType::KEYWORD_ARG, "False", "axis will not be initially fit to data extents on the first rendered frame" });
+        args.push_back({ mvPyDataType::Bool, "no_menus", mvArgType::KEYWORD_ARG, "False", "the user will not be able to open context menus with right-click" });
+        args.push_back({ mvPyDataType::Bool, "no_side_switch", mvArgType::KEYWORD_ARG, "False", "the user will not be able to switch the axis side by dragging it" });
+        args.push_back({ mvPyDataType::Bool, "no_highlight", mvArgType::KEYWORD_ARG, "False", "the axis will not have its background highlighted when hovered or held" });
+        args.push_back({ mvPyDataType::Bool, "opposite", mvArgType::KEYWORD_ARG, "False", "axis ticks and labels will be rendered on the conventionally opposite side (i.e, right or top)" });
+        args.push_back({ mvPyDataType::Bool, "foreground_grid", mvArgType::KEYWORD_ARG, "False", "grid lines will be displayed in the foreground (i.e. on top of data) instead of the background" });
+        args.push_back({ mvPyDataType::String, "tick_format", mvArgType::KEYWORD_ARG, "''", "Sets a custom tick label formatter" });
+        args.push_back({ mvPyDataType::Integer, "scale", mvArgType::KEYWORD_ARG, "internal_dpg.mvPlotScale_Linear", "Sets the axis' scale. Can have only mvPlotScale_ values"});
+        args.push_back({ mvPyDataType::Bool, "log_scale", mvArgType::DEPRECATED_KEYWORD_ARG, "False", "Old way to set log scale in the axis. Use 'scale' argument instead.", "See the new scale argument." });
+        args.push_back({ mvPyDataType::Bool, "time", mvArgType::DEPRECATED_KEYWORD_ARG, "False", "Old way to set time scale in the axis. Use 'scale' argument instead.", "See the new scale argument." });
+        args.push_back({ mvPyDataType::Bool, "axis_opposite", mvArgType::KEYWORD_ARG, "False", "the axis will be inverted" });
```

Comment:
This is not what I asked for :cry:. Moverover, this change doesn't really make the code any better.

[My comments](https://github.com/hoffstadt/DearPyGui/pull/2275#pullrequestreview-2039815628) (_first two comments by that link, click "Show resolved"_) were about `invert` and `opposite` on `mvColorMapScale`, not on `mvPlotAxis`.

The reason I was commenting on `invert` is that `mvColorMapScale` is a color-related widget, and when we're talking about colors, the word "invert" is typically related to inversion of the color (black -> white, green -> purple, etc.).

In case of `opposite`, the reason was a bit different. When I see `plot_axis(opposite=True)`, I'd imagine the axis to be opposite to something or on the opposite side of something (and it's really on the opposite side of the plot). This is fine and doesn't need to change.

However, when I see `add_colormap_scale(opposite=True)`, I'd imagine the _colormap scale widget_ to be opposite to something or on the opposite side of something - and it is not! Instead, it's rendered as if it were mirrored (BTW `mirror` might be a good name for it).

Reply:
Now it should be okay, I changed `axis_opposite` -> `opposite` and (`mvColorMapScale`)`opposite` -> `mirror`
Instead `invert` was already changed to `reverse_dir`

Reply:
The problem with `invert_order` is that the old version already has it under the name `invert`, so the change would break existing apps. Let's keep it as `invert` in `mvPlotAxis` parser.

Everything else looks good :+1:.